pkg/openwsn/Makefile

```
@@ -13,6 +13,9 @@ IGNORE_MODULES  := openwsn_leds \
                    openwsn_iee802154e_security \
                    openwsn_radio \
                    openwsn_riotos \
+                   openwsn_sock_udp \
+                   openwsn_sock \
+                   openwsn_sock_async \
```

```suggestion
                   openwsn_sock \
                   openwsn_sock_async \
                   openwsn_sock_udp \
```

let's keep alphabetical order

---

addressed